libraries/AP_AdvancedFailsafe/AP_AdvancedFailsafe.cpp

```diff
@@ -277,6 +283,11 @@ AP_AdvancedFailsafe::check(uint32_t last_valid_rc_ms)
         } else if (gcs_link_ok) {
             _state = STATE_AUTO;
             gcs().send_text(MAV_SEVERITY_DEBUG, "AFS State: AFS_AUTO, GCS now OK");
+
+            if(option_is_set(OPTION::CONTINUE_AFTER_RECOVERED)){
```

Comment:
```suggestion
            if(option_is_set(Option::CONTINUE_AFTER_RECOVERED)){
```

Reply:
Could you explain why you recommend this change? I'm not sure if this is correct.

Reply:
Perhaps it should be AdvancedFailsafe::OPTION::CONTINUE_AFTER_RECOVERED ?

Reply:
> Perhaps it should be AdvancedFailsafe::OPTION::CONTINUE_AFTER_RECOVERED ?

No, this is simply a stylistic thing; we usually name our `enum class`es like this

Reply:
FYI I recently added enum class naming to the wiki: https://ardupilot.org/dev/docs/style-guide.html#enums